test/bdd/fixtures/fabric/config/ledger/did-sidetree-protocol-v0_1.json

```diff
@@ -1,14 +1,15 @@
 {
   "genesisTime": 1,
   "multihashAlgorithms": [18],
-  "maxOperationCount": 5000,
+  "maxOperationCount": 10,
```

Comment:
Why this change?

Reply:
copy from sidetree-fabric bddtest config

Reply:
Let's not change it to a small number. @fqutishat

Reply:
also looks like the filename didn't get renamed from 0_1 @sandrask

Reply:
What version would you like 1_0?

Reply:
@fqutishat We also need to change sidetree/0.0.1 to sidetree/v1 in config files

Reply:
done